doc/stdlib/dune

```diff
@@ -0,0 +1,17 @@
+; This is an ad-hoc rule to easy the migration, it should be handled
+; natively by Dune in the future.
+(rule
+  (targets html)
+  (deps
+   (source_tree %{project_root}/theories)
+   (source_tree %{project_root}/plugins)
+   (:header %{project_root}/doc/common/styles/html/coqremote/header.html)
+   (:footer %{project_root}/doc/common/styles/html/coqremote/footer.html))
+ (action
+  (progn
+   (bash "mkdir -p html")
```

Comment:
A bit curious why you need bash here. `(run mkdir -p html)` doesn't work?

Reply:
Good point, thanks I never know if this kind of commands are shell builtins or not.